easybuild/easyconfigs/x/XALT/CC_config.py

```diff
@@ -0,0 +1,272 @@
+#                     [ t_1,     probability_1],
+#                     ...
+#                     [ t_n,     probability_n],
+#                     [ 1.0e308, 1.0],
+#
+#
+# The first number is the left edge of the time range.  The
+# second number is the probability of being sampled. Where a
+# probability of 1.0 means a 100% chance of being recorded and a
+# value of 0.01 means a 1% chance of being recorded.
+#
+# So a table that looks like this:
+#     interval_array = [
+#                       [ 0.0,                0.0001 ],
+#                       [ 300.0,              0.01   ],
+#                       [ 600.0,              1.0    ],
+#                       [ sys.float_info.max, 1.0    ]
+#     ]
+#
+# would say that program with execution time that is between
+# 0.0 and 300.0 seconds has a 0.01% chance of being recorded.
+# Execution times between 300.0 and 600.0 seconds have a 1%
+# chance of being recorded and and programs that take longer
+# than 600 seconds will always be recorded.
+#
+# The absolute minimum table would look like:
+#
+#     interval_array = [
+#                       [ 0.0,                1.0 ],
+#                       [ sys.float_info.max, 1.0 ]
+#     ]
+#
+# which says to record every scalar (non-mpi) program no matter
+# the execution time.
+#
+# Note that scalar execution only uses this table IFF
+# $XALT_SAMPLING equals yes
+
+
+interval_array = [
+    [    0.0,              0.0001 ],
+    [ 1800.0,              0.01   ],
+    [ 7200.0,              1.0    ],
+    [ sys.float_info.max,  1.0    ]
+]
+
+#------------------------------------------------------------
+# Sites can also define a different sampling specification
+# for mpi programs different from interval_array.  If no
+# mpi_interval_array is given then the interval_array is used
+# for both scalar and mpi programs.
+
+mpi_interval_array = [
+    [    0.0,              0.0001 ],
+    [  900.0,              0.01   ],
+    [ 1800.0,              1.0    ],
+    [ sys.float_info.max,  1.0    ]
+]
+
+#------------------------------------------------------------
+# XALT filter environment variables.  Those variables
+# which pass through the filter are save in an SQL table that is
+# searchable via sql commands.  The environment variables are passed
+# to this filter routine as:
+#
+#      env_key=env_value
+#
+# So the regular expression patterns must match the whole string.
+
+
+# The value on the left is either KEEP or SKIP.  If the value
+# is KEEP then if the environment string matches the regular
+# expression then the variable is stored. If the value on the left
+# is SKIP then if the variable matches it is not stored.
+
+# Order of the list matters.  The first match is used even if a
+# later pattern would also match.  The upshot is that special pattern
+# matches should appear first and general ones later.
+
+# If the environment string does not match any pattern then it is
+# marked as SKIP.
+
+
+env_patterns = [
+    # These are monitored
+
```

Comment:
Everything we have decided to monitor can be found somewhere else in the record. I decided to keep everything that is skipped so it can easily be added later if need be.